src/main/resources/entity-types/invoice/drv_invoice_details.json5

```diff
@@ -0,0 +1,508 @@
+    },
+    {
+      name: 'payment_id',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "invce.jsonb->>'paymentId'",
+    },
+    {
+      name: 'disbursement_date',
+      dataType: {
+        dataType: 'dateType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "invce.jsonb->>'disbursement_date'",
+    },
+    {
+      name: 'po_numbers',
+      dataType: {
+        dataType: 'arrayType',
+        itemDataType: {
+          dataType: 'stringType',
+        },
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "(\n  SELECT\n    array_agg(alc_id.value::text) FILTER (\n      WHERE\n        (alc_id.value::text) IS NOT NULL\n    ) AS ARRAY_AGG\n  FROM\n    jsonb_array_elements_text(invce.jsonb->'poNumbers'::text) alc_id (value)\n)",
+      filterValueGetter: "(\n  SELECT\n    array_agg(lower(alc_id.value::text)) FILTER (\n      WHERE\n        (alc_id.value::text) IS NOT NULL\n    ) AS ARRAY_AGG\n  FROM\n    jsonb_array_elements_text(invce.jsonb->'poNumbers'::text) alc_id (value)\n)",
+      valueFunction: 'lower(:value)',
+    },
+    {
+      name: 'vendor_id',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "invce.jsonb->>'vendorId'",
+    },
+    {
+      name: 'fiscal_year_id',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "invce.jsonb->>'fiscalYearId'",
+    },
+    {
+      name: 'account_no',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "invce.jsonb->>'accountNo'",
+    },
+    {
+      name: 'manual_payment',
+      dataType: {
+        dataType: 'booleanType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "invce.jsonb->>'manualPayment'",
+      values: [
+        {
+          label: 'True',
+          value: 'true',
+        },
+        {
+          label: 'False',
+          value: 'false',
+        },
+      ],
+    },
+    {
+      name: 'acq_unit_ids',
+      dataType: {
+        dataType: 'arrayType',
+        itemDataType: {
+          dataType: 'rangedUUIDType',
+        },
+      },
+      queryable: false,
+      visibleByDefault: false,
+      valueGetter: "(\n  SELECT\n    array_agg(acq_id.value::text) FILTER (\n      WHERE\n        (acq_id.value::text) IS NOT NULL\n    ) AS ARRAY_AGG\n  FROM\n    jsonb_array_elements_text(invce.jsonb->'acqUnitIds'::text) acq_id (value)\n)",
+      filterValueGetter: "(\n  SELECT\n    array_agg(lower(acq_id.value::text)) FILTER (\n      WHERE\n        (acq_id.value::text) IS NOT NULL\n    ) AS ARRAY_AGG\n  FROM\n    jsonb_array_elements_text(invce.jsonb->'acqUnitIds'::text) acq_id (value)\n)",
```

Comment:
Do we need this filterValueGetter? It looks like it may be the same as the valueGetter